### ISSUE TITLE
Educational Worlds Filter Only Shows Exact Match for "Educational"

### ISSUE THREAD

**Issue by lxw404** [Yellow-Dog-Man/Resonite-Issues#2461]
### Describe the Underlying Issue

Currently the Educational worlds filter tab only seems to look for the exact "educational" search term, which makes it seem that there are very few of these worlds, when in reality many of them are just tagged with other synonymous terms such as:
* education
* edu
* esd
* science
* edusci

### To Reproduce

Open the Educational tab and see that it mostly excludes every MMC ESD entry.

### Expected Behavior

A new user searching for educational worlds might not see the very excellent MMC entries and other worlds which are not tagged explicitly as "Educational", so this tab should probably be more flexible with its search terms.

### Screenshots / Video

5 ESD entries included in Educational:
![Resonite_2MmNOLp0e4](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/52231149/65fbd74c-3f38-44a5-a4e6-a8c3e89f40ec)

Total 31 ESD entries:
![Resonite_N3rYIE0V0f](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/52231149/a7c3c9b3-2d7d-43f9-b6c2-79ff3a0b7e12)

### Additional Context

This is something I recently noticed when [asked for good Educational worlds by Ryan Schultz on discord](https://discord.com/channels/1040316820650991766/1258066909031829555).

### Reporters

LuxKitty

**Comment by shiftyscales** [comment]
For now this has been by design- the way the world presets work is by setting various data sources, search parameters, etc.

E.g. for educational, the preset looks like this.

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/27816116-0b16-4dae-b8c4-0594c1bad127)

While it is possible to modify the base search- when searching directly for tags, currently you can only search for worlds containing all of the given tags- not other operations to search for any of the tags.

E.g. if I modify the base search to include "+edu" this is the result I get instead.

![image](https://github.com/Yellow-Dog-Man/Resonite-Issues/assets/54213390/6aa68921-8b48-406d-971f-44f59e1b843e)

Seeking input from @Frooxius - with the new worlds datafeed, would it be possible for the content team to build the new world UI in a way that arbitrary / compound tag searching could be done like this?

Or would it still require additional work to the underlying search mechanisms?

**Comment by Frooxius** [comment]
@shiftyscales No, the worlds data feed doesn't let you list published worlds yet.

However this is an easy change to make with this - we can just change data preset to include multiple terms, instead of just the one.
